scripts/check_ipfs_hashes_pushed.py

```diff
@@ -23,23 +23,25 @@
 import json
 import subprocess  # nosec
 import sys
-from typing import Optional
+from concurrent.futures import ThreadPoolExecutor
+from typing import Optional, Tuple
 
 import requests
 
 
 IPFS_ENDPOINT = "https://gateway.autonolas.tech/ipfs"
+MAX_WORKERS = 10
 
 
-def check_ipfs_hash_pushed(ipfs_hash: str) -> bool:
+def check_ipfs_hash_pushed(ipfs_hash: str) -> Tuple[str, bool]:
     """Check that the given ipfs hash exists in the registry"""
 
     try:
         url = f"{IPFS_ENDPOINT}/{ipfs_hash.strip()}"
-        res = requests.get(url, timeout=120)
-        return res.status_code == 200
+        res = requests.get(url, timeout=5)
```

Comment:
120 was indeed too large, though please comment more extensively because such changes are hard to detect otherwise. Also a constant would be nice.